### PR TITLE
update log to directpv

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

**Review by harshavardhana — DISMISSED** [review]

File: pkg/client/init.go [hunk 1]
```diff
@@ -47,6 +50,19 @@ var (
 	metadataClient                 metadata.Interface
 	latestDirectCSIDriveInterface  directcsi.DirectCSIDriveInterface
 	latestDirectCSIVolumeInterface directcsi.DirectCSIVolumeInterface
+
+	binaryName = func() string {
```
Comment (wlan0) [inline pkg/client/init.go:54]: Can you move these two to the utils package. We shouldn't have the same definitions in two different places.

**Review by wlan0 — APPROVED** [review]
LGTM

File: pkg/converter/drive_downgrade.go [hunk 2]
```diff
@@ -75,7 +75,7 @@ func driveDowngradeV1Beta1ToV1alpha1(unstructured *unstructured.Unstructured) er
 		return err
 	}
 
-	klog.V(10).Infof("Converting directcsidrive: %v to v1alpha1", v1beta1DirectCSIDrive.Name)
+	klog.V(10).Infof("Converting directpvdrive: %v to v1alpha1", v1beta1DirectCSIDrive.Name)
```
Comment (Praveenrajmani) [inline pkg/converter/drive_downgrade.go:78]: we have not yet moved to directpvdrive crd. we can call it `directcsidrive` only.

File: pkg/converter/volume_downgrade.go [hunk 3]
```diff
@@ -75,7 +75,7 @@ func volumeDowngradeV1Beta1ToV1alpha1(unstructured *unstructured.Unstructured) e
 		return err
 	}
 
-	klog.V(10).Infof("Converting directcsivolume: %v to v1alpha1", v1beta1DirectCSIVolume.Name)
+	klog.V(10).Infof("Converting directpvvolume: %v to v1alpha1", v1beta1DirectCSIVolume.Name)
```
Comment (Praveenrajmani) [inline pkg/converter/volume_downgrade.go:78]: we have not yet moved to directpvvolume crd. we can call it directcsivolume only.

**Review by wlan0 — APPROVED** [review]

**Pull request merged**